docs/GettingStartedDocs/Contributors/WindowsSGX1FLCGettingStarted.md

```diff
@@ -39,8 +39,9 @@ Run the following from PowerShell to deploy all the prerequisites for building O
 ./scripts/install-windows-prereqs.ps1
 ```
 
-On Windows 10, the Intel PSW is delivered via Windows Update. Running the
-executable installer will fail on Windows 10 machines. To skip PSW installation:
+On Windows 10, the Intel PSW is delivered via Windows Update, and on Windows Server 2019, the Intel
+PSW is already included. Running the executable installer will fail on Windows 10 and Windows Server,
+unless we skip PSW installation. To skip PSW installation:
```

Comment:
Consider clarifying:
``` suggestion
On Windows Server 2019 and versions of Windows 10 newer than 1709, the Intel PSW
should already be automatically installed. Attempting to run the PSW installer will fail if
that is the case. To skip the PSW installer:
```
As with the comment in WindowsManualSGX1FLCDCAPPrereqs.md, it might be useful to include a pointer for individuals to the manual installation or upgrade instructions if this is not the case on their devices (or they want to manually update the driver/components). If we're document the use of the `PSW_INF_RS3_and_above` folder, consider adding a reference here (and WindowsSGX1GettingStarted.md) like: